docs-compose/overwatch_docs/content/DataEngineer/Pipeline_Management.md

```diff
@@ -120,6 +104,47 @@ For the developers -- This output is created as a DataSet via the Case Class
+
+### PipReport Structure
+[**SAMPLE**](/assets/TableSamples/pipreport.tab)
+
+**KEY** -- organization_id + moduleID + Overwatch_RunID
+
+**Write Mode** -- Append
+
+| Column          | Type      | Description                                                                                                                                                                                                             |
+|:----------------|:----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| organization_id | string    | Canonical workspace id                                                                                                                                                                                                  |
+| workspace_name  | string    | Customizable human-legible name of the workspace, should be globally unique within the organization                                                                                                                     | 
+| moduleId        | int       | Module ID -- Unique identifier for the module                                                                                                                                                                           |
+| moduleName      | string    | Name of the module                                                                                                                                                                                                      |
+| primodialDate   | date      | The date from which Overwatch will capture the details. The format should be yyyy-MM-dd ex: 2022-05-20 == May 20 2022                                                                                                   | 
+| fromTS          | long      | The snapshot start time from which data will be loaded for the module                                                                                                                                                   |
+| untilTS         | long      | The snapshot final time from which data will be loaded for the module. This is also the epoch millis of Pipeline_SnapTS                                                                                                 |
+| status          | string    | Terminal Status for the module run (SUCCEEDED, FAILED, ROLLED_BACK, EMPTY, etc). When status is failed, the error message and stack trace that can be obtained is also placed in this field to assist in finding issues |
+| write_metrics   | string    | Contains write metric for a particular chunk of data written to a table. It has information like number of files, Output Rows and Output bytes being written.                                                           |  
```

Comment:
Maybe mention this will only be populated if the module's data is written in a Merge. Append only tables won't have metrics here

Reply:
As per my understanding this write_metric is available irrespective of the write mode